src/main/java/org/apache/maven/plugins/jlink/JLinkMojo.java

```diff
@@ -468,7 +468,12 @@ private List<File> getCompileClasspathElements(MavenProject project) {
         List<File> list = new ArrayList<>(project.getArtifacts().size() + 1);
 
         for (Artifact a : project.getArtifacts()) {
-            getLog().debug("Artifact: " + a.getGroupId() + ":" + a.getArtifactId() + ":" + a.getVersion());
+            boolean skip = "pom".equals(a.getType());
```

Comment:
is "pom" available somehwere as constant?

Reply:
> is "pom" available somehwere as constant?

not really